server.go

```diff
@@ -128,6 +128,10 @@ func (s *Server) Broadcast(msg string, except *Client) {
 			}
 			client.Send(personalMsg)
 		} else {
+			if client.quietMode && strings.HasPrefix(msg, systemMessageFormat) {
```

Comment:
This will strip out messages that just happen to have similar formatting as systemMessageFormat (e.g. from users with the same color).

Also how about making it an `else if` and keeping the `else` as-is?